codice/alliance
# CAL-33 Implement an image chipping service

#### What does this PR do?
Adds the chipping service and a chipping MetacardTransformer.

#### Who is reviewing it?
@jaymcnallie 
@kcwire 
@peterhuffer 
@beyelerb 

#### How should this be tested?
1) build it
2) ingest a NITF
3) search for the NITF
4) get the resource URI and the metacard location
5) construct a WKT that lies within the metacard location
5) use a broswer to request the chip: 
    Pattern: /services/catalog/sources/ddf.distribution/<id>?transform=chip&location=<wkt>
    Example: 
6) verify that the resulting image is the area of the image defined by the wkt.

#### Any background context you want to provide?
#### What are the relevant tickets?
CAL-33

#### Screenshots (if appropriate)

![chip-ss](https://cloud.githubusercontent.com/assets/5922390/14870778/6c6d43d0-0c91-11e6-80af-c477bed3a833.png)

#### Checklist:
